site/en/docs/privacy-sandbox/aggregation-service/index.md

```diff
@@ -124,9 +116,33 @@ generating summary reports for the
 [Private Aggregation API](/docs/privacy-sandbox/summary-reports#private-aggregation) 
 and the [Attribution Reporting API](/docs/privacy-sandbox/summary-reports#attribution-reporting).
 
+## Test the Aggregation Service {: #test}
+
+We recommend reading the corresponding experiment and participate guide for the API you're testing:
+
+* [Attribution Reporting API](/docs/privacy-sandbox/attribution-reporting-experiment/)
+* [Private Aggregation API](/docs/privacy-sandbox/private-aggregation-experiment/)
+
+### Local testing
+
+We've created a local testing tool to process aggregatable reports for Attribution Reporting and the Private Aggregation API. [Read the instructions](https://github.com/privacysandbox/aggregation-service/blob/main/README.md).
+
+To use this tool, install a Java Runtime Environment.
```

Comment:
I don't think this line is necessary since it's already in the instructions that's being linked, and the setup steps are more involved than just setting up a JRE.

Reply:
Removed!